salt/matchers/compound_pillar_exact_match.py

```diff
@@ -14,10 +14,13 @@
 log = logging.getLogger(__name__)
 
 
-def mmatch(expr, delimiter, greedy):
+def mmatch(expr, delimiter, greedy, opts=None):
     '''
     Return the minions found by looking via pillar
     '''
-    ckminions = salt.utils.minions.CkMinions(__opts__)
+    if not opts:
+        ckminions = salt.utils.minions.CkMinions(__opts__)
+    else:
+        ckminions = salt.utils.minions.CkMinions(opts)
```

Comment:
can we just make this `ckminions = salt.utils.minions.CkMinions(opts or __opts__)` and get rid of the rest of the if statement?